src/python/aqueduct_executor/operators/systemmetric_executor/main.py

```diff
@@ -0,0 +1,37 @@
+import argparse
+import base64
+import traceback
+import sys
+
+from aqueduct_executor.operators.utils import enums, utils
+from aqueduct_executor.operators.systemmetric_executor import spec
+from aqueduct_executor.operators.utils.storage.parse import parse_storage
+
```

Comment:
To confirm my understanding, all this system metric executor does is:
1) Pull the specific metric out of the output of the previous artifact's metadata path.
2) Write that metric out to the output path, not the metadata path.

Reply:
Thats correct, the output metadata of the previous OP func is fed to this executor, this then extracts the information it needs from the previous op run and stores it as the output content of the metric artifact that its tied the output to.